backend/src/main/java/com/soen341/instagram/service/impl/ProfileService.java

```diff
@@ -53,6 +54,10 @@ public void setEmail(final String email)
 	public void setBiography(final String biography)
 	{
 		final Account account = getCurrentAccount();
+		if (biography.length() > 150)
+		{
+			throw new BiographyLengthTooLongException("Biography length cannot exceed 150 characters");
+		}
```

Comment:
Not necessary but I would just refactor to BiographyTooLongException, current name seems too long and it would be just as descriptive